pkg/orchestrator/orchestrator.go

```diff
@@ -78,7 +78,12 @@ func NewOrchestratorSpecificConfigProvider() (OrchestratorSpecificConfigProvidin
 	case AzureDevOps:
 		return &AzureDevOpsConfigProvider{}, nil
 	case GitHubActions:
-		return &GitHubActionsConfigProvider{}, nil
+		ghProvider := &GitHubActionsConfigProvider{}
+		// Temporary workaround: The orchestrator provider is not always initialized after being created,
+		// which causes a panic in some places for GitHub Actions provider, as it needs to initialize
+		// github sdk client.
+		ghProvider.InitOrchestratorProvider(&OrchestratorSettings{})
```

Comment:
Why is InitOrchestratorProvider implemented everywhere but not used?

Reply:
Internally it's called once.

Reply:
In most of the places where `NewOrchestratorSpecificConfigProvider` is called, it's okay for the provider to be uninitialized because they use methods that fetch data from environment variables. However, in a `telemetry` package, the provider doesn't get initialized after calling `NewOrchestratorSpecificConfigProvider`, and then it attempts to call the `GetJobURL` method, which causes a panic. This occurs because the `GetJobURL` method requires an initialized github client:
https://github.com/SAP/jenkins-library/blob/20b3c0d7280442b773f69acdd77f4755e68ba474/pkg/telemetry/telemetry.go#L47-L53
https://github.com/SAP/jenkins-library/blob/20b3c0d7280442b773f69acdd77f4755e68ba474/pkg/telemetry/telemetry.go#L90-L93
I'm not sure if that case happens in inner-source commands, didn't check it.